Podcast/PlayerControlsView.swift

```diff
@@ -14,40 +14,49 @@ protocol PlayerControlsDelegate: class {
     func playerControlsDidTapSkipBackward()
     func playerControlsDidScrub()
     func playerControlsDidEndScrub()
+    func playerControlsDidTapMoreButton()
+    func playerControlsDidTapRecommendButton()
 }
 
 class PlayerControlsView: UIView {
     
-    let forwardsBackwardsButtonSize: CGFloat = 44
-    let playPauseButtonSize: CGFloat = 40
-    let sliderHeight: CGFloat = 6
-    let sliderInset: CGFloat = 48
-    let sliderCenterY: CGFloat = 25
-    let timeLabelHorizontalInset: CGFloat = 12
-    let controlButtonsCenterY: CGFloat = 71.5
-    let controlButtonsCenterHorizontalOffset: CGFloat = 55
+    let sliderHeight: CGFloat = 1.5
+    let marginSpacing: CGFloat = 18.5
     
-    let playerControlsViewHeight: CGFloat = 116
+    let playerControlsViewHeight: CGFloat = 192
+    
+    let playPauseButtonSize: CGSize = CGSize(width: 72, height: 72)
+    let skipButtonSize: CGSize = CGSize(width: 27, height: 29)
+    let skipButtonSpacing: CGFloat = 40.5
+    let skipButtonYInset: CGFloat = 79
+    let sliderYInset: CGFloat = 132
+    let timeLabelSpacing: CGFloat = 8
+    let buttonsYInset: CGFloat = 181.5
+    
+    let recommendButtonSize: CGSize = CGSize(width: 80, height: 18)
+    let moreButtonSize: CGSize = CGSize(width: 25, height: 18)
     
     var slider: UISlider!
     var playPauseButton: UIButton!
     var forwardsButton: UIButton!
-    var forwardsLabel: UILabel!
     var backwardsButton: UIButton!
-    var backwardsLabel: UILabel!
     var rightTimeLabel: UILabel!
     var leftTimeLabel: UILabel!
+    var recommendButton: RecommendButton!
+    var moreButton: MoreButton!
     
     weak var delegate: PlayerControlsDelegate?
     
     override init(frame: CGRect) {
         super.init(frame: frame)
         self.frame.size.height = playerControlsViewHeight
-        backgroundColor = .white
+        backgroundColor = UIColor.colorFromCode(0xf4f4f7)
```

Comment:
put this is Utilities if UIColor+ extension as a podcast color (it's probably already in there if not add it)